test/index.tsx

```diff
@@ -50,7 +50,12 @@ test({
 `.trim()
 
     const marp = mount(<Marp markdown={markdown} />)
-    expect(marp.text().trim()).toBe(
+    expect(
+      marp
+        .text()
+        .replace(/!function.*;/, '')
```

Comment:
This replacement is to remove the polyfill code. (I believe the tests should ignore the polyfill code)

Reply:
Do you mean using `<Marp markdown={markdown} options={{ script: false }} />` in the tests? Or forcing to change `script` option to `false` in `Marp.tsx` ?

Reply:
And Marp Core peerDependency will require >= v1.0.1.
https://github.com/marp-team/marp-react/blob/ab45842487a08cef0057a367422d909ae051fbfe/package.json#L93